apps/web-mzima-client/src/app/settings/surveys/create-task-modal/create-task-modal.component.html

```diff
@@ -39,7 +39,11 @@
     </mat-form-field>
   </div>
   <div class="toggle">
-    <mat-slide-toggle [(ngModel)]="newTask.required" [data-qa]="'task-toggle-required'">
+    <mat-slide-toggle
+      [(ngModel)]="newTask.required"
+      (keydown.enter)="newTask.required = !newTask.required"
+      [data-qa]="'task-toggle-required'"
+    >
       {{ 'survey.require_section' | translate }}
```

Comment:
I **think** but I am not entirely sure, that we should add space here as well? Both space and enter activates controls like buttons, dropdowns etc...

Reply:
@Angamanga you're right, both space and enter activate controls but in some use-cases, space is use to scroll the page. For consistency, should we stick to using enter to activate controls while space for scrolling?

Reply:
Yes, lets keep it like it is 👍 We could always add it later if we see a need.

Reply:
Okay, thank you @Angamanga